docs/introduction/how-to-run.rst

```diff
@@ -41,12 +41,16 @@ This is the main command we'll run Auspice with, as it makes Auspice available i
 | narrativeDir        | PATH                     | Directory where narratives (Markdown files) are sourced. This is ignored if you define custom handlers. |
 +---------------------+--------------------------+---------------------------------------------------------------------------------------------------------+
 
-For more complicated setups, where you define your own server handlers, see `supplying custom handlers to the Auspice server <../server/api.md#supplying-custom-handlers-to-the-auspice-server>`__.
+For more complicated setups, where you define your own server handlers, see :ref:`supplying custom handlers to the Auspice server <server-api-supplying-custom-handlers>`.
 
 ``auspice build``
 -----------------
 
-Build the client source code bundle. This is needed in three cases: 1. You have installed Auspice from source, or updated the source code. 1. You are editing the source code and need to rebuild the client 1. You wish to build a customised version of the Auspice client. See `Customising Auspice <../customise-client/index>`__ for more info.
+Build the client source code bundle. This is needed in three cases:
+
+1. You have installed Auspice from source, or updated the source code.
+2. You are editing the source code and need to rebuild the client
+3. You wish to build a customised version of the Auspice client. See `Customising Auspice <../customise-client/index>`__ for more info.
```

Comment:
This link needs updating

Reply:
Thanks, I've found a few others going through all occurrences of

    >`__

Will fix.

Reply:
Fixed in [force-push](https://github.com/nextstrain/auspice/compare/f0fda9dd7e139f8a7b12ee1a444104f8288e6873..a178f04c1ab889f9f577886c5f027d47a189b45b).